less/index.less

```diff
@@ -25,152 +92,284 @@ time, mark, audio, video {
 	vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
 	display: block;
 }
+
 body {
 	line-height: 1;
 }
-ol, ul {
+
+ol,
+ul {
 	list-style: none;
 }
-blockquote, q {
+
+blockquote,
+q {
 	quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
+
+blockquote:after,
+blockquote:before,
+q:after,
+q:before {
 	content: '';
 	content: none;
 }
+
 table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
-
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+body,
+html {
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
 }
 
-h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 10px;
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-family: 'Bangers', cursive;
+	letter-spacing: 1px;
+	margin-bottom: 10px;
 }
 
 .container {
-    width: 880px;
-    margin: auto;
+	max-width: 880px;
+	margin: 0 auto;
+	padding: 0 20px;
+	@media (max-width: 768px) {
+		max-width: 768px;
+	}
+	@media (max-width: 400px) {
+		max-width: 400px;
+	}
 }
+// variables
```

Comment:
Here is a good spot for global styles (header color, nav color, etc.) which have other variables assigned to them, like you did with @nav-link-color-hover, and then use the color website to better name the colors: http://chir.ag/projects/name-that-color/.

So for example, you could have:
`@nav-link-color: @silver-chalice;`

`@silver-chalice: #b0b0b0`

Then you can use `@nav-link-color` everywhere, and if you ever want to change it, you can change it at the top. So the structure is to have your theme variables, and then a bunch of colors below it.

Reply:
@EricHech thanks for the feedback!